### PR TITLE
Add scale ci templates

### PR DESCRIPTION
**Pull request by jkilpatr**
These are the current templates being used for OpenShift on OpenStack
scale CI in the lab.

### CODE REVIEW

File: RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+parameter_defaults:
+  NeutronExternalNetworkBridge: "''"
+  InternalApiNetCidr: 172.16.0.0/24
+  TenantNetCidr: 172.17.0.0/24
+  StorageNetCidr: 172.18.0.0/24
+  StorageMgmtNetCidr: 172.19.0.0/24
+  ManagementNetCidr: 172.20.0.0/24
+  ExternalNetCidr: 172.21.0.0/24
```
Comment (smalleni) [inline RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml]: Change the CIDRs here...maybe /16.

File: RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  ExternalNetCidr: 172.21.0.0/16
+  InternalApiAllocationPools: [{'start': '172.16.0.10', 'end': '172.16.0.200'}]
+  TenantAllocationPools: [{'start': '172.17.0.10', 'end': '172.17.0.200'}]
+  StorageAllocationPools: [{'start': '172.18.0.10', 'end': '172.18.0.200'}]
+  StorageMgmtAllocationPools: [{'start': '172.19.0.10', 'end': '172.19.0.200'}]
+  ManagementAllocationPools: [{'start': '172.20.0.10', 'end': '172.20.0.200'}]
+  ExternalAllocationPools: [{'start': '172.21.0.10', 'end': '172.21.0.100'}]
+  # Set to the router gateway on the external network
+  ExternalInterfaceDefaultRoute: 172.21.0.1
+  PublicVirtualFixedIPs: [{'ip_address':'172.21.0.10'}]
```
Comment (mbruzek) [inline RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml]: We are going to default to a 172.21.0.0/22 cidr, which gives us a range of 1022 ip addresses 172.21.0.0-172.21.3.255. Should this value be 172.21.0.2 (leaving 1 for the gateway)? http://www.subnet-calculator.com/cidr.php

File: RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  StorageNetCidr: 172.18.0.0/16
+  StorageMgmtNetCidr: 172.19.0.0/16
+  ManagementNetCidr: 172.20.0.0/16
+  ExternalNetCidr: 172.21.0.0/16
+  InternalApiAllocationPools: [{'start': '172.16.0.10', 'end': '172.16.0.200'}]
+  TenantAllocationPools: [{'start': '172.17.0.10', 'end': '172.17.0.200'}]
+  StorageAllocationPools: [{'start': '172.18.0.10', 'end': '172.18.0.200'}]
+  StorageMgmtAllocationPools: [{'start': '172.19.0.10', 'end': '172.19.0.200'}]
+  ManagementAllocationPools: [{'start': '172.20.0.10', 'end': '172.20.0.200'}]
+  ExternalAllocationPools: [{'start': '172.21.0.10', 'end': '172.21.0.100'}]
```
Comment (mbruzek) [inline RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml]: This pool should be 172.21.0.2 through 172.21.3.255 with the new /22 cidr

File: RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+
+parameter_defaults:
+  NeutronExternalNetworkBridge: "''"
+  InternalApiNetCidr: 172.16.0.0/16
+  TenantNetCidr: 172.17.0.0/16
+  StorageNetCidr: 172.18.0.0/16
+  StorageMgmtNetCidr: 172.19.0.0/16
+  ManagementNetCidr: 172.20.0.0/16
+  ExternalNetCidr: 172.21.0.0/16
```
Comment (mbruzek) [inline RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml]: Does this need to change with the cidr change on the subnet?

**Comment by mbruzek** [comment]
@smalleni I realize /16 gives more room, but do these ranges need to match in order for this to work?

**Comment by smalleni** [comment]
Not necessarily, I wanted to have /16 there just in case you want to scale beyond 250+ hosts (each baremetal host consumes one IP in each of these subnets). It is not related to the issue you are facing. 


 The only two things that need to match AFAIK is the CIDR of the floating ip subnet pool you create and the CIDR here https://github.com/jkilpatr/tripleo-quickstart-scalelab/blob/43c81ea8948ae7ff98ab2004e939517e9d887e48/profiles/main.yml#L52

**Comment by mbruzek** [comment]
I updated the PR to default to the /16 cidr to match this change:  https://github.com/redhat-performance/tripleo-quickstart-scalelab/pull/1/files

**Comment by mbruzek** [comment]
@jkilpatr We need to change the allocation pools to larger so we have more floating IP addresses to pull from. I think this also needs to be in the next deploy.

File: RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  InternalApiAllocationPools: [{'start': '172.16.0.10', 'end': '172.16.0.200'}]
```
Comment (mbruzek) [inline RDU-Scale/Ocata/openshift-scalelab-ci/network-environment.yaml]: All these pools need to be expanded. start: 172.16.0.3, end: 172.21.255.254

**Pull request merged**